### ISSUE TITLE
Contact forms not working on 4.5.1

### ISSUE THREAD

**Issue by magnoliaman** [wpsharks/comet-cache#753]
It seems that my sites that use Contact Form 7 with WordPress 4.5.1 and Comet Cache are not displaying any page content if it has a contact form. I deactivate Comet Cache and all is well. I'm also using Google's Recaptcha on most (maybe all) of these pages. It doesn't seem to matter what theme I'm using. Other forms don't appear to be impacted with this problem.

Here's an example: https://www.shoppingbargains.ca/contact/

**Comment by raamdev** [comment]
@magnoliaman I just installed the latest version of Contact Form 7 (v4.4.2) and Comet Cache (v160417), on a test site running WordPress v4.5.2 and the TwentyFifteen theme. 

I created a Contact Form with the `[recaptcha]` placeholder to enable the captcha, placed it on a test page, and then logged out and viewed the page. It showed up as expected (and Comet Cache reported in the HTML Notes that it was not caching the page—as expected—because it contains `nonce` values, which are not cache-compatible).

I'm marking this as `cannot reproduce`. The only thing I can think might be causing a problem for you is your theme or another plugin that you have installed.
